src/migration/tasks/MigrateSimplePricing.ts

```diff
@@ -0,0 +1,66 @@
+import PricingDefinition, { PricingEntity } from '../../types/Pricing';
+import { PricingSettingsType, SimplePricingSetting } from '../../types/Setting';
+import Tenant, { TenantComponents } from '../../types/Tenant';
+
+import Constants from '../../utils/Constants';
+import Logging from '../../utils/Logging';
+import MigrationTask from '../MigrationTask';
+import PricingStorage from '../../storage/mongodb/PricingStorage';
+import { ServerAction } from '../../types/Server';
+import SettingStorage from '../../storage/mongodb/SettingStorage';
+import TenantStorage from '../../storage/mongodb/TenantStorage';
+import Utils from '../../utils/Utils';
+
+const MODULE_NAME = 'MigrateSimplePricing';
+
+export default class SimplePricingMigrationTask extends MigrationTask {
+  async migrate(): Promise<void> {
+    const tenants = await TenantStorage.getTenants({}, Constants.DB_PARAMS_MAX_LIMIT);
+    for (const tenant of tenants.result) {
+      await this.migrateTenant(tenant);
```

Comment:
You can also check if the simple pricing is active for the tenant, but the setting should suffice.